source/GlassView.Core/Models/BenchmarkCase.cs

```diff
@@ -0,0 +1,19 @@
+namespace Atmoos.GlassView.Core.Models;
+
+public sealed record class BenchmarkCase : IName
+{
+    public required String Name { get; init; }
+    public required Boolean IsBaseline { get; init; }
+    public required String[] Categories { get; init; }
+    public required Parameter[] Parameters { get; init; }
+    public required String HardwareIntrinsics { get; init; }
+    public required StatisticsInfo Statistics { get; init; }
+    public required AllocationInfo Allocation { get; init; }
+}
+
+public sealed record class Parameter
```

Comment:
Should we keep one class / record per file, or do you think it's not that important?
Or perhaps the Parameter should be an internal record of the BenchmarkCase?

Reply:
Thanks. I had missed that one :-)